rainier-tests/src/test/scala/com/stripe/rainier/core/DiscreteTest.scala

```diff
@@ -0,0 +1,188 @@
+package com.stripe.rainier.core
```

Comment:
This test file has been removed from the Develop branch, not sure why. I can take out the `check` test and leave just the test I wrote, if you want.

Reply:
It may have been taken out because it's assumed that `SBCTest` replaces it. Can you check if https://github.com/stripe/rainier/blob/develop/rainier-tests/src/test/scala/com/stripe/rainier/core/SBCTest.scala#L23 (which has been commented out) works now?

Reply:
It doesn't, but if it was being generated off of an incorrect goldset, maybe we need to update the goldset? (I'm not familiar with these sbc tests!)

Reply:
yeah, you need to `run` instead of `test`. Right now it's showing overdispersion (which is consistent with what you found). Hopefully when you run it, it looks more uniform.

![Screen Shot 2019-03-12 at 5 50 01 PM](https://user-images.githubusercontent.com/9892/54245771-642c2d00-44ef-11e9-855c-2aff106b6d74.png)

Reply:
sorry, my screenshot was over-cropped, but note that you have to be in `project rainierTests`

Reply:
Uhh, 
![image](https://user-images.githubusercontent.com/33037397/54247082-d5baaa00-44f4-11e9-8613-5b28829e3e45.png)
the bottom line overflowed and made the others look odd. Basically the bottom line hugely overflowed, how unfortunate.

Reply:
Ok, so it still doesn't "pass" that test...

Reply:
Specifically it's showing that when you generate samples from the Binomial with a true value of `x`, and then try to infer `x'` from those samples, your inferred `x'` is biased to be lower than `x`.

Reply:
If we presume that the `logDensity` and hence the inference is correct, that would imply that our generation is now biased to be too low.

Reply:
Strange that it's now a one-sided error

Reply:
@mikeheaton-stripe should we discuss in https://gitter.im/com_stripe_rainier/Lobby ?

Reply:
:blue-portal: